sdk/mediaservices/Microsoft.Azure.Management.Media/tests/ScenarioTests/LiveEventTests.cs

```diff
@@ -12,7 +12,7 @@ namespace Media.Tests.ScenarioTests
 {
     public class LiveEventTests : MediaScenarioTestBase
     {
-        [Fact]
+        [Fact(Skip = "Need to rerun for 2019-05-01-preview")]
```

Comment:
Hi Brian, as public preview is still shipped to all the customers, we need to ensure its quality aka proper test coverage. Wouldn't want to skip them

Reply:
@isra-fel, the tests that are marked skip don't test the new feature. We want to skip them to save the feature team from having to rerecord the test with APIs that use the version 2019-05-01-preview and then rerecord them again when we move the feature into the stable version - 2018-07-01, regardless if they choose to update the test with new API calls.
The new feature that we're previewing in 2019-05-01-preview has been thoroughly tested with both the .NET SDK and the Java SDK.

Reply:
@isra-fel, I have removed the skip attribute and rerecorded the tests. Please review.
Thanks